### PR TITLE
fix: Query to the media store may throws an exception

### PR DESCRIPTION
**Pull request by makingthematrix**
Happening on Android 8.0 and 8.1 on some less popular devices (Redmi, Xperia).
We try to access the media store to check if there are already sound files for our notifications.
If they are not, we want to put our own custom audio files there.
But on some device the query causes Android to throw `SQLiteException` which is not handled.

The fix handles the exception, treating it as if there was no audio files in the media store.

**This is a hotfix**: In the future we should test this behaviour on a phone which crashes the app,
and fix the actual source of the bug.
#### APK
[Download build #12205](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12205/artifact/build/artifact/wire-dev-PR1921-12205.apk)
[Download build #12206](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12206/artifact/build/artifact/wire-dev-PR1921-12206.apk)

### CODE REVIEW

File: app/src/main/scala/com/waz/zclient/notifications/controllers/NotificationManagerWrapper.scala [hunk 1]
```diff
@@ -29,6 +30,7 @@ import android.support.v4.app.NotificationCompat.Style
 import android.support.v4.app.{NotificationCompat, RemoteInput}
 import android.text.style.{ForegroundColorSpan, StyleSpan}
 import android.text.{SpannableString, Spanned}
+import com.waz.ZLog
```
Comment (CalumMcCall) [inline app/src/main/scala/com/waz/zclient/notifications/controllers/NotificationManagerWrapper.scala]: Maybe use ZLog2?

File: app/src/main/scala/com/waz/zclient/notifications/controllers/NotificationManagerWrapper.scala [hunk 2]
```diff
@@ -404,18 +408,28 @@ object NotificationManagerWrapper {
         contentValues.put(MediaStore.Audio.AudioColumns.IS_NOTIFICATION, true)
         contentValues.put(MediaStore.Audio.AudioColumns.IS_ALARM, true)
         contentValues.put(MediaStore.Audio.AudioColumns.IS_MUSIC, false)
-        val uri = MediaStore.Audio.Media.INTERNAL_CONTENT_URI
 
-        val query = cxt.getContentResolver.query(uri, null, s"${MediaStore.MediaColumns.DATA} LIKE '%$name%'", null, null)
-        if (query.getCount == 0) {
-          cxt.getContentResolver.insert(uri, contentValues)
+        // TODO: On some devices (Redmi 6A, Xperia X Compact) the query causes SQLiteException.
+        // test on one of these devices and find out why.
```
Comment (CalumMcCall) [inline app/src/main/scala/com/waz/zclient/notifications/controllers/NotificationManagerWrapper.scala]: Maybe create a ticket somewhere so we don't forget this?

Comment (makingthematrix) [reply app/src/main/scala/com/waz/zclient/notifications/controllers/NotificationManagerWrapper.scala]: https://github.com/wireapp/android-project/issues/352

File: app/src/main/scala/com/waz/zclient/notifications/controllers/NotificationManagerWrapper.scala (hunk 1 above)
Comment (makingthematrix) [reply app/src/main/scala/com/waz/zclient/notifications/controllers/NotificationManagerWrapper.scala]: I'm unable to use ZLog2 for this - the uri which I want to display in the error message does not implement `showSafe`. I changed the ZLog.error a bit, though.

**Pull request merged**